src/popup.tsx

```diff
@@ -159,14 +193,60 @@ const Popup = () => {
                 setTypes(newTypes);
               }}
             />
-
+            <div
+              ref={(node) => {
+                colorsRef.current[idx] = node;
+              }}
+              className={`bg-${colors[idx]}-500 w-4 h-4`}
+              style={{
+                backgroundColor: colors[idx],
+                flexShrink: 0,
+              }}
+            ></div>
+            <svg
```

Comment:
I don't think using svg directly in the tsx file is a good idea. Can u try use the svg as an image instead?